application-admintools-ui/src/main/resources/AdminTools/WebHome.xml

```diff
@@ -161,6 +164,233 @@ $services.admintools.getConfigurationData('configuration')
+        <displayFormType>select</displayFormType>
+        <displayType>yesno</displayType>
+        <name>parse</name>
+        <number>4</number>
+        <prettyName>Parse content</prettyName>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.BooleanClass</classType>
+      </parse>
+      <use>
+        <cache>0</cache>
+        <disabled>0</disabled>
+        <displayType>select</displayType>
+        <freeText>forbidden</freeText>
+        <largeStorage>0</largeStorage>
+        <multiSelect>0</multiSelect>
+        <name>use</name>
+        <number>3</number>
+        <prettyName>Use this extension</prettyName>
+        <relationalStorage>0</relationalStorage>
+        <separator> </separator>
+        <separators>|, </separators>
+        <size>1</size>
+        <unmodifiable>0</unmodifiable>
+        <values>currentPage|onDemand|always</values>
+        <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
+      </use>
+    </class>
+    <property>
+      <cache>default</cache>
+    </property>
+    <property>
+      <code>require(['jquery'], function($) {
+  $(document).on('click', '#filesDownloadFilesModal .btn-primary', function(event) {
+    event.preventDefault()
+    var downloadForm = $(document.getElementById("filesDownloadForm"));
+    var form = downloadForm.serialize()
+    var link = document.createElement('a');
+    link.href = downloadForm.attr('action');
+    link.href += "?" + form;
+    link.click();
+  });
+
+    $(document).on('click', '#filesViewLastNLinesModal .btn-primary', function(event) {
+    event.preventDefault()
+    var downloadForm = $(document.getElementById("filesLastNLogLinesForm"));
+    var noLines = $("#filesNoLines").val();
+    if(noLines == ""){
+      noLines = "1000";
+    }
+    var link = document.createElement('a');
+    link.href = downloadForm.attr('action');
+      debugger;
+    link.href += "/" + noLines;
+    link.target="_blank";
+    link.click();
+  });
+
+  $(document).on('click', '#configurationViewLastNLinesModal .btn-primary', function(event) {
+    event.preventDefault()
+    var downloadForm = $(document.getElementById("configurationLastNLogLinesForm"));
+    var noLines = $("#configurationNoLines").val();
+    if(noLines == ""){
+      noLines = "1000";
+    }
+    var link = document.createElement('a');
+    link.href = downloadForm.attr('action');
+      debugger;
+    link.href += "/" + noLines;
+    link.target="_blank";
+    link.click();
+  });
+  
+  $(document).ready(function() {
+    var logsCheckbox = $('input[name="files"][value="logsDataResource"]');
+    var dateFields = $('.dateFields');
+
+    if (!logsCheckbox.is(':checked')) {
+        dateFields.hide();
+    } else {
+        dateFields.show();
+    }
+
+    logsCheckbox.change(function() {
+        if (this.checked) {
+            dateFields.show();
+        } else {
+            dateFields.hide();
+        }
+    });
+});
```

Comment:
I think this can be refactored, check https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/delete.js#L21 for something similar